themes/digital.gov/layouts/partials/core/guides/guide-prev-next.html

```diff
@@ -1,48 +1,34 @@
-{{- $guide_data := index $.Site.Data.guidenav -}}
+{{- $guideData := index $.Site.Data.guidenav -}}
 {{ if .Params.guide }}
-  {{- $guide_data = index $.Site.Data.guidenav (.Params.guide) -}}
+  {{- $guideData = index $.Site.Data.guidenav (.Params.guide) -}}
 {{ end }}
 
-{{- if $guide_data.showNextPrevious -}}
-  {{- $current_page := . -}}
-  {{- $current_url := $current_page.Permalink -}}
+{{- if $guideData.showNextPrevious -}}
+  {{- $currentPage := . -}}
+  {{- $currentURL := $currentPage.Permalink -}}
 
-  {{- $guide_files := (where .Site.Pages ".Params.guide" .Params.guide) -}}
+  {{- $prevItem := 0 -}}
+  {{- $pointItem := 0 -}}
 
-  {{- $prev_item := 0 -}}
-  {{- $point_item := 0 -}}
 
-
-  <div class="dg-guide__guide-buttons grid-row">
-    {{- range $guide_data.nav -}}
+  <div class="dg-guide__prev-next grid-row">
+    {{- range $guideData.nav -}}
       {{- $item := . -}}
 
-      {{- $prev_item = $point_item -}}
-      {{- $point_item = $item -}}
+      {{- $prevItem = $pointItem -}}
+      {{- $pointItem = $item -}}
 
-      {{- if not (eq $prev_item 0) -}}
+      {{- if not (eq $prevItem 0) -}}
```

Comment:
It would help for future reference to add a comment above the `if` block with a short explanation of what it does.